examples/src/main/java/com/yugabyte/examples/UniformLoadBalanceExample.java

```diff
@@ -11,15 +11,15 @@
 import java.util.*;
 
 public class UniformLoadBalanceExample {
-  private static String controlUrl = "";
-  private static HikariDataSource hikariDataSource;
+  public static String controlUrl = "";
```

Comment:
Why public? It should be protected instead. Right?

Reply:
yes, thanks